src/proto/mgmt/pool.proto

```diff
@@ -153,3 +141,15 @@ message PoolSetPropResp {
 	}
 }
 
+// PoolQueryResp represents a pool query response.
+message PoolQueryResp {
+	int32 status = 1; // DAOS error code
+	string uuid = 2; // pool uuid
+	uint32 totaltargets = 3; // total targets in pool
+	uint32 activetargets = 4; // active targets in pool
+	uint32 disabledtargets = 5; // number of disabled targets in pool
+	PoolRebuildStatus rebuild = 6; // pool rebuild status
+	StorageUsageStats scm = 7; // SCM storage usage stats
+	StorageUsageStats nvme = 8; // NVMe storage usage stats
+}
+
```

Comment:
Seems odd to have this after the PoolSetProp messages. Bad merge? Not something that needs to be fixed, but if you need to adjust anything it would be good to fix this.

Reply:
Yeah, I had a conflict to resolve in this file after changing the disabled field and rebasing on the set-prop patch. I can move it back up if I have to correct anything else.